client/src/components/Sub_RelatedItems/styles/related.css

```diff
@@ -53,13 +52,16 @@
 .single-prod {
   flex-direction: column;
   margin-inline: auto;
-  /* border: 2px solid #636363; */
   border-radius: var(--littleRound);
   padding: 0.8rem;
   margin: 0.5rem;
   box-shadow: rgba(14, 30, 37, 0.12) 0px 2px 4px 0px, rgba(14, 30, 37, 0.32) 0px 2px 16px 0px;
 }
 
+.single-prod.container.focused {
+  box-shadow: rgba(14, 30, 37, 0.12) 0px 2px 4px 0px, rgba(235, 69, 95, 0.58) 6px 2px 16px 0px;
```

Comment:
love the combined CSS selector + the new box shadow. you must teach me your ways